WalletWasabi/Services/CcjCoordinator.cs

```diff
@@ -257,7 +257,7 @@ private async Task<int> AdjustConfirmationTargetAsync(bool lockCoinJoins)
 				int confirmationTarget = RoundConfig.ConfirmationTarget.Value;
 				for (int i = 0; i < unconfirmedCoinJoinsCount; i++)
 				{
-					confirmationTarget /= 2;
+					confirmationTarget = (int)(confirmationTarget * 0.7);
```

Comment:
I think it would be a good idea to put the `0.7` as configurable setting because we don't know if we will need to tweak it again in the future.

Reply:
Great idea! I'll do that.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/1457/commits/16dd7fcbc8747f021a2db2b444c5fa29c0d0af5a